test/TestSkipImportPlaybook.py

```diff
@@ -24,26 +23,18 @@
 '''
 
 
-class TestSkipBeforeImport(unittest.TestCase):
-    def setUp(self):
-        rulesdir = os.path.join('lib', 'ansiblelint', 'rules')
-        self.rules = RulesCollection([rulesdir])
-
-        # make dir and write role tasks to import or include
-        self.play_root = tempfile.mkdtemp()
-        with open(os.path.join(self.play_root, 'imported_playbook.yml'), 'w') as f_main:
-            f_main.write(IMPORTED_PLAYBOOK)
-
-    def tearDown(self):
-        shutil.rmtree(self.play_root)
+@pytest.fixture
+def playbook(tmpdir):
```

Comment:
[`tmp_path` fixture] should be used instead since `tmpdir` is supposed to go away as per https://github.com/ansible/ansible-lint/pull/740#issuecomment-617896072.

[`tmp_path` fixture]:
https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture

Reply:
(and I see you already use the proper `tmp_path` fixture in another place in this PR)